Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-5554.dfy.expect

```diff
@@ -0,0 +1 @@
+false
```

Comment:
same here. I think the expect files are immune to trailing whitespaces, but I would double check.